### PR TITLE
fix e2e-telco5g

### PR DESCRIPTION
**Pull request by yuvalk**
trying to understand why job is still failing on PULL_BASE_REF

### CODE REVIEW

**Comment by yuvalk** [comment]
@sshnaidm please review

File: ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-commands.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ fi
 echo "************ telco5g gather-pao commands ************"
 
 pao_mg_tag="4.10" # pao must-gather does not have 'latest' tag - setting 4.10 as a workaround for now.
+PULL_BASE_REF=${PULL_BASE_REF:-master"}
```
Comment (sshnaidm) [inline ci-operator/step-registry/telco5g/gather-pao/telco5g-gather-pao-commands.sh:21]:
```suggestion
PULL_BASE_REF=${PULL_BASE_REF:-"master"}
```

**Review by sshnaidm — CHANGES_REQUESTED** [review]

**Comment by yuvalk** [comment]
/retest
missing cli image isnt caused by us

**Pull request merged**